pkg/provider/handlers/functions.go

```diff
@@ -92,7 +93,7 @@ func GetFunction(client *containerd.Client, name string, namespace string) (Func
 
 	spec, err := c.Spec(ctx)
 	if err != nil {
-		return Function{}, fmt.Errorf("unable to load function spec for reading secrets: %s, error %w", name, err)
+		return Function{}, fmt.Errorf("unable to load function spec for reading secrets and limits: %s, error %w", name, err)
```

Comment:
I'm confused by this error message, especially what we had before.

I think this should just be: `unable to load function %s, error: %w`

Reply:
Sure, I understand, changed.